org.mwc.cmap.core/src/org/mwc/cmap/core/DebriefToolParent.java

```diff
@@ -160,6 +160,9 @@ public void logError(final int status, final String text, final Exception e)
 	{
 		CorePlugin.logError(status, text, e);
 		//prompt Error Log view to user up on error report.
+		if( PlatformUI.getWorkbench()!=null 
+		    &&  PlatformUI.getWorkbench().getActiveWorkbenchWindow()!=null 
+		    &&  PlatformUI.getWorkbench().getActiveWorkbenchWindow().getActivePage()!=null)
 		try
     {
       PlatformUI.getWorkbench().getActiveWorkbenchWindow().getActivePage().showView("org.eclipse.pde.runtime.LogView");

```

Comment:
Hmm, the Error Log doesn't open for me.  The second "if" test is always failing.  But, I know I have an Active Workbench Window available.  I expect it's because it's being called from a non-UI thread.